js/controller.js

```diff
@@ -497,6 +497,17 @@ module.exports = function(OO, _, $, W) {
       this.renderSkin();
     },
 
+    toggleAirPlayIcon: function(event) {
+      if (!this.state.airPlayStatusIcon) {
+        this.state.airPlayStatusIcon = 'airPlay-disconnected';
```

Comment:
Don't we want to render after this as well?

Reply:
Added renderSkin